CHANGES.rst

```diff
@@ -55,6 +55,9 @@ datamodels
 
 - Add the ``P_READPA`` keyword to the ``ReadnoiseModel`` schema [#6973]
 
+- Change name of ``P_READPA`` keyword in datamodel metadata to ``p_readpatt``
+  to be consistent with other pattern keyword names [#7001]
```

Comment:
Technically, the "p" stands for "parameterized", as opposed to "pattern".

Reply:
Ah, I went off of this: https://jwst-crds.stsci.edu/static/users_guide/reference_conventions.html#id2

Reply:
Hmmm, OK. Well, if it's referred to that way in the docs, that's fine. Not a big deal either way for the change log.